packages/react-aria-components/src/Popover.tsx

```diff
@@ -150,9 +150,13 @@ function PopoverInner({state, isExiting, UNSTABLE_portalContainer, ...props}: Po
 
   let style = {...popoverProps.style, ...renderProps.style};
 
+  let onPointerDown = () => {
+    state.close();
```

Comment:
won't this affect a lot more than menus?
popovers were already closing without this, I assume that was interact outside?

all popovers are dismissable, so I don't think this will hurt other things, but I'd like to have this specifically called out for testing

additionally, should we split this into pointerDown for mouse but pointerUp for touch? and do we need the mouse/touch fall backs? keeping in mind it'd really be onPress for touch, the event sequence would need to start and end on the underlay so that a swipe or scroll on the popover doesn't accidentally close it.

I'm fine for the touch stuff to be followup, but we do need to test this

Reply:
We do this for RSP Popovers:

https://github.com/adobe/react-spectrum/blob/ee51290b98fdc7cad31957ed2dd0a486eb0c1427/packages/%40react-spectrum/overlays/src/Popover.tsx#L120-L122

I think everything you mentioned makes sense. I'll bring this up with the team soon.